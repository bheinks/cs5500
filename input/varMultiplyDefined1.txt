(*
	Variable is multiply defined in different declaration lines
*)

program p;

var x : integer;
    y : boolean;
    x : char;

begin
  write(5)
end.

