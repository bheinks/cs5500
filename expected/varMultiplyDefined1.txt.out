

>>> Entering new scope...

+++ Adding p to symbol table with type PROGRAM

+++ Adding x to symbol table with type INTEGER

+++ Adding y to symbol table with type BOOLEAN

+++ Adding x to symbol table with type CHAR
Line 9: Multiply defined identifier
